### PR TITLE
rc: dev to canay

### PR DESCRIPTION
**Pull request by archx3**

### CODE REVIEW

File: src/task_queue/models.py [hunk 1]
```diff
@@ -55,10 +57,11 @@ class Task(models.Model):
         to=UserProfile
     )
     last_run = models.DateField(blank=True, null=True)
+    is_archived = models.BooleanField(default=False)
```
Comment (BradHN1) [inline src/task_queue/models.py:60]: Do we want this model update?  Doesn't hurt anything but the decision was made not to use it.

File: src/api/store/community.py [hunk 2]
```diff
@@ -1288,6 +1290,26 @@ def update_community_notification_settings(self, context, args) -> Tuple[dict, M
...
             
+            if activate_on:
+                more_info = notification_setting.more_info or {}
+                task_id = more_info.get("task_id")
+                if task_id:
+                    AsyncResult(task_id).revoke()
+                
+                eta = datetime.strptime(activate_on, "%Y-%m-%d") + timedelta(minutes=0)
+                
+                task_id = automatically_activate_nudge.apply_async(args=[notification_setting.id], eta=eta).id
```
Comment (BradHN1) [inline src/api/store/community.py:1301]: Question - queued task survive when a new version of API is deployed? I think so but want to be sure.

Comment (abdullai-t) [reply src/api/store/community.py:1301]: I am not too sure about it but I think they do survive

File: src/task_queue/tasks.py [hunk 3]
```diff
@@ -1,52 +1,71 @@
...
+
+def get_task_info(task):
+	if task:
+		return f"task: {task.name}, job: {task.job_name}, task_id: {task.id}"
+	
+def log_status(task, message):
+	task_name = get_task_info(task)
 
-ENV = "API Local" if IS_LOCAL else "API CANARY" if IS_CANARY else "API PROD" if IS_PROD else "API DEV"
+	if task.status == FAILED:
```
Comment (BradHN1) [inline src/task_queue/tasks.py:27]: someones linter isn't going to like these tab indentations.   Better to set them right the first time.

File: src/Makefile [hunk 4]
```diff
@@ -139,3 +139,7 @@ migrate-canary:
 migrate-local:
 	DJANGO_ENV=local python manage.py migrate
 .PHONY: migrate-local
+
+
+create-cache-table-local:
```
Comment (BradHN1) [inline src/Makefile:144]: Is this used by anything?  if not should remove it.

File: src/_main_/settings.py [hunk 5]
```diff
@@ -207,6 +207,13 @@
 #     }
 # }
 
+CACHES = {
```
Comment (BradHN1) [inline src/_main_/settings.py:210]: is this used?  if not remove.

Comment (abdullai-t) [reply src/_main_/settings.py:210]: George has some incoming changes that will use it

File: src/task_queue/models.py (hunk 1 above)
Comment (abdullai-t) [reply src/task_queue/models.py:60]: We might need it in the future, so let's keep it for now.

**Pull request merged**